examples-kotlin/src/main/kotlin/org/bitcoinj/ForwardingService.kt

```diff
@@ -24,63 +24,58 @@ import org.bitcoinj.wallet.Wallet
 import org.bitcoinj.wallet.listeners.WalletCoinsReceivedEventListener
 import java.io.Closeable
 import java.io.File
-import java.util.Objects.*
+import java.util.*
```

Comment:
We generally try to avoid using wildcard imports...

Reply:
will look out those